packages/manager/apps/telecom/src/app/telecom/telephony/service/voicemail/management/management.html

```diff
@@ -260,5 +278,39 @@
         ></div>
     </div>
     <!-- /.widget-presentation -->
+    <div data-ng-if="VoicemailManagementCtrl.isOpenModal">
+        <div class="modal d-block">
+            <div
+                class="modal-dialog modal-dialog-centered modal-dialog-scrollable"
+            >
+                <div class="modal-content">
+                    <form>
```

Comment:
No needed, because you have a modal in modal 
(oui-modal it's already a modal)

Reply:
Indeed you don't have routing in this place.
Because oui-modal need:
views: {
        modal: {
          component: 'xxxxxxxx',
        },
      },
      layout: 'modal',
      
      in routing. but not possible in your case